srcpkgs/nginx/template

```diff
@@ -93,8 +99,13 @@ pre_build() {
 	sed -i 's/-lcrypt/$(LDFLAGS) &/' objs/Makefile
 }
 
+do_check() {
+	TEST_NGINX_BINARY="$XBPS_BUILDDIR/${pkgname}-${version}/objs/nginx" \
+		prove "../${pkgname}-tests-${_tests_commit}"
+}
+
 post_install() {
-	sed -e 's|\<user\s\+\w\+;|user html;|g' \
+	sed -e 's|\<user\s\+\w\+;|user http;|g' \
```

Comment:
Could you explain why you decided to change this?

Reply:
Uh, i will not change it actually. I haven't rebase it yet.